### DISCUSSION TITLE
Add BCU to Winget

### DISCUSSION THREAD

**Discussion by ETHREAL1** [Klocman/Bulk-Crap-Uninstaller#410] [Ideas]
Hello, Was directed to this by a coworker in IT. I use Winget to basically install and update many systems. would be nice if BCU was available on Winget. That is all.

**Comment by Klocman** [comment]
If it's not up there yet, feel free to add BCU to the winget repository https://github.com/microsoft/winget-pkgs#submitting-a-package

**Reply by ETHREAL1** [reply]
It's usually better if the dev adds it, That and I have never added a repository to Winget and don't know what is involved with that. I can try to but...

**Comment by Klocman** [comment]
It has been added.

**Comment by uyriq** [comment]
please update to [@latest](https://github.com/Klocman/Bulk-Crap-Uninstaller/releases/tag/v5.8.1) version
